### ISSUE TITLE
Replace optional NPM dependency webworker-threads with web-worker

### ISSUE THREAD

**Issue by onlyurei** [NaturalNode/apparatus#17]
Looks like [webworker-threads](https://github.com/audreyt/node-webworker-threads) is not maintained anymore, and the author recommends to use [web-worker](https://www.npmjs.com/package/web-worker) instead:
https://github.com/audreyt/node-webworker-threads/issues/204
https://github.com/audreyt/node-webworker-threads/issues/206

**Comment by Hugo-ter-Doest** [comment]
I don't see a dependency here on webworker-threads. I think you mean the dependency in NaturalNode, here:
https://github.com/NaturalNode/natural/blob/master/lib/natural/classifiers/classifier_train_parallel.js
Is that what you mean?

I can replace it with webworker.

**Comment by onlyurei** [comment]
> I don't see a dependency here on webworker-threads. I think you mean the dependency in NaturalNode, here: https://github.com/NaturalNode/natural/blob/master/lib/natural/classifiers/classifier_train_parallel.js Is that what you mean?
> 
> I can replace it with webworker.

@Hugo-ter-Doest Yes, sorry I opened the issue in the wrong repo - was trying to understand the e2e flow of NB classification class.

**Comment by Hugo-ter-Doest** [comment]
I will ook into this.
